recipes-extended/xen/files/blktap3-build-integration.patch

```diff
@@ -1,35 +1,22 @@
-diff --git a/tools/Makefile b/tools/Makefile
-index c903d6a63e..d0c6daaea2 100644
 --- a/tools/Makefile
 +++ b/tools/Makefile
-@@ -20,7 +20,7 @@ SUBDIRS-y += console
+@@ -19,6 +19,7 @@ SUBDIRS-$(CONFIG_X86) += firmware
+ SUBDIRS-y += console
  SUBDIRS-y += xenmon
  SUBDIRS-y += xenstat
- SUBDIRS-$(CONFIG_Linux) += memshr 
--SUBDIRS-$(CONFIG_BLKTAP2) += blktap2
 +SUBDIRS-$(CONFIG_BLKTAP2) += blktap3
```

Comment:
Is this stuff still needed? blktap3 is a separate repo, you've already rm'd the xen-tools-blktap recipes, does this and the ```XEN_BLKTAP2``` need to make it into Xen's tools Makefile?

Reply:
It is removed in:
commit d3189b5328c3c9362d90058d8efcf59c14825f93
Author: Jason Andryuk <jandryuk@gmail.com>
Date:   Fri Nov 19 14:16:04 2021 -0500

    Build blktap3 outside xen

Reply:
ack thanks. I'm going through these commit-by-commit.